containers/unit-tests/start

```diff
@@ -16,6 +18,12 @@ while test $# -gt 0; do
     shell) flags=-it; cmd=/bin/bash;;
     :*) image=ghcr.io/cockpit-project/unit-tests"$1";;
     distcheck|check-memory) scenario="--env=TEST_SCENARIO=$1";;
+    dist)
+        scenario="--env=TEST_SCENARIO=$1"
+        results_vol="--volume ${top_srcdir}/tmp/dist:/results:Z"
+        mkdir -m 777 -p "${top_srcdir}/tmp/dist"
```

Comment:
Can't say I find this to be particularly wonderful...

I start to think we really need to change the way we invoke the container.  Our simple `run` approach is getting increasingly brittle.  I feel like we might split it into a more general create/copy/start/copy type of approach...

That can be saved for another day, though.